liquibase.yaml

```diff
@@ -60,6 +85,94 @@ pipeline:
       tar -zxvf liquibase-dist/target/${{package.name}}-${{package.version}}.tar.gz -C ${{targets.destdir}}/usr/share/java/liquibase
       ln -sf /usr/lib/jvm/default-jvm ${{targets.destdir}}/opt/java/openjdk
 
+  - name: Check verification markers
+    runs: |
+      # There was an issue where the following jar wasn't being included. Explicitly ensure that it is included
+      test -f ${{targets.destdir}}/usr/share/java/liquibase/internal/lib/commons-collections4.jar
+
+      # Upstream also has intended files lists, validate using those
+      echo "==> Validating distribution contents against expected files"
+      # List tarball contents, excluding directories (which end with /)
+      tar -tzf liquibase-dist/target/${{package.name}}-${{package.version}}.tar.gz | grep -v '/$' | sort > actual-contents.txt
+
+      # Check against the standard expected contents
+      echo "Comparing with expected-distribution-contents.txt"
+      # Filter out commercial components from expected files since we build with !liquibase-commercial
+      grep -v 'liquibase-commercial.jar' liquibase-dist/expected-distribution-contents.txt | sort > expected-filtered.txt
+
+      if diff -u expected-filtered.txt actual-contents.txt; then
```

Comment:
This `diff`'s output will be written to the log, so I don't know if we need the `comm` calls below.